templates/archives/doc.jinja2

```diff
@@ -70,3 +83,20 @@
     </div>
 
 {% endblock %}
+
+{% block scripts %}
+    <script>
+        function toggle_citation() {
+            const citation = document.getElementById('citation');
```

Comment:
@samimak37 - at your leisure you might want to look into how to rewrite this using jQuery (since we're already using that on this project). The syntax is nice and terse.